code/addons/interactions/src/constants.ts

```diff
@@ -1,2 +1,5 @@
 export const ADDON_ID = 'storybook/interactions';
 export const PANEL_ID = `${ADDON_ID}/panel`;
+
+export const TUTORIAL_VIDEO_LINK = 'https://youtu.be/Waht9qq7AoA';
+export const DOCUMENTATION_LINK = 'writing-stories/play-function';
```

Comment:
Given that the title of the empty state is "Interaction testing", should this be:

```suggestion
export const DOCUMENTATION_LINK = 'writing-tests/interaction-testing';
```

Reply:
It could be! the play function page just felt a little more digestible. It got to the point quicker.

Technically `writing-tests/interaction-testing` would be more informative though 🤔

Reply:
Let's go with the testing one, please.